core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -46,7 +44,7 @@ object TransactionMarkerChannelManager {
             txnStateManager: TransactionStateManager,
             txnMarkerPurgatory: DelayedOperationPurgatory[DelayedTxnMarker],
             time: Time): TransactionMarkerChannelManager = {
-
+    val logContext = new LogContext("[TransactionMarkerChannelManager brokerId=" + config.brokerId + "] ")
```

Comment:
We could pass this to `TransactionMarkerChannelManager` so that it can be used to set `logIdent` (instead of duplicating the logic). We'd have to add a `logPrefix` method to `LogContext`.

Reply:
@ijuma should I also replace Logger usage which uses `logIdent` with LogContext?

Reply:
I suggest not doing that in this PR. We can open a separate JIRA about that to discuss the best way to do it. My suggestion was to use the same log prefix for `logIdent` and `logContext` and the easiest way to do that is to create the `LogContext` first and then pass its value to `logIdent`. Does that make sense?

Reply:
Yes, it makes total sense to move it to another ticket.